### PR TITLE
Docs for optimizer

### PR DESCRIPTION
**Pull request by RodionovF**

### CODE REVIEW

File: include/deepworks/optimizers.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#### Attributes
+- `lr` – step size
+- `gamma` – exponential decay rate for the velocity
+vector (default: 0.9)
+#### Example
+```cpp
+float lr = 1e-3f;
+float gamma = 0.95f;
+dw::optimizer::SGDMomentum opt(model.params(), lr, gamma);
+// dw::optimizer::SGDMomentum opt(model.params(), lr);
```
Comment (TolyaTalamanov) [inline include/deepworks/optimizers.md:32]: Why it's commented ?

**Review by TolyaTalamanov — APPROVED** [review]
LGTM :+1:

**Comment by TolyaTalamanov** [comment]
@l-bat

**Review by l-bat — APPROVED** [review]

File: include/deepworks/optimizers.md (hunk 1 above)
Comment (RodionovF) [reply include/deepworks/optimizers.md:32]: i want to show, that user can pass all the parameters, or use the default ones and leave only model.params and learning rate

Comment (TolyaTalamanov) [reply include/deepworks/optimizers.md:32]: A bit confusing, but OK let it be

**Pull request merged**